Core/Utilities/ElasticSearchDB.py

```diff
@@ -25,6 +27,20 @@
 
 sLog = gLogger.getSubLogger(__name__)
 
+
+def ifConnected(method):
```

Comment:
I really do not think that this is necessary. 
You can just actually attempt the search, the insert or whatever. And if it is not connected, you'll get a `ConnectionError` exception

Reply:
Yes, but normally we don't let exceptions raise in DIRAC, while we have `S_OK` or `S_ERROR`. No discussion on this point, please! ;)